eth/stagedsync/stage_mining_create_block.go

```diff
@@ -288,7 +302,13 @@ func SpawnMiningCreateBlockStage(s *StageState, tx kv.RwTx, cfg MiningCreateBloc
 	}
 
 	current.Header = header
-	current.Uncles = makeUncles(env.uncles)
+	if isTrans {
+		current.Header.Time = cfg.presetFields.Timestamp
+		current.Header.MixDigest = cfg.presetFields.Random
+		current.Uncles = nil
+	} else {
+		current.Uncles = makeUncles(env.uncles)
```

Comment:
Do we need this if we are just going to get rid of uncles entirely for POS?

Reply:
we just discard them completely for now, in the future we will remove all Uncle logic